lib/msal-common/src/cache/entities/CredentialEntity.ts

```diff
@@ -126,14 +131,19 @@ export class CredentialEntity {
         clientId: string,
         realm?: string,
         target?: string,
-        familyId?: string
+        familyId?: string,
+        tokenType?: string
```

Comment:
Reason for using type string here instead of the enum?

Reply:
@tnorling  The AT with AuthScheme spec points out that the `tokenType` value is to be stored as-is from the server response. Would that prevent us from making it an enum? Otherwise I think it's a good idea.

Reply:
You're comparing it to an enum below. I'm assuming that means if it isn't a pre-defined, known value i.e. "pop" that we're discarding it? Do I have that right?